mimic/model/behaviors.py

```diff
@@ -167,3 +176,14 @@ def behavior_for_attributes(self, attributes):
             if criteria.evaluate(attributes):
                 return behavior
         return self.event.default_behavior
+
+    def remove_behavior_by_id(self, behavior_id):
+        """
+        Remove a previously-registered behavior given the behavior's ID.
+        """
+        for i, behaviors in enumerate(self.registered_behaviors):
+            b, c, b_id = behaviors
+            if b_id == behavior_id:
+                self.registered_behaviors.pop(i)
```

Comment:
stylistically I would just use `del self.registered_behaviors[i]` instead of `pop` since you're not using the result